content/id/docs/concepts/containers/container-environment-variables.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Variabel Environment Kontainer
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Laman ini menjelaskan berbagai *resource* yang tersedia di dalam Kontainer pada suatu *environment*.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## *Environment* Kontainer
+
+*Environment* Kontainer pada Kubernetes menyediakan beberapa *resource* penting yang tersedia di dalam Kontainer:
+
+* Sebuah *Filesystem*, yang merupakan kombinasi antara [image](/docs/concepts/containers/images/) dan satu atau banyak [*volumes*](/docs/concepts/storage/volumes/).
+* Informasi tentang Kontainer tersebut.
+* Informasi tentang objek-objek lain di dalam kluster.
```

Comment:
in the future we should use objek instead of obyek as mentioned 👍

Reply:
noted